_targets_eval_postprocessing.R

```diff
@@ -604,7 +630,7 @@ manuscript_figures <- list(
       date_to_plot = forecast_date_to_plot
     )
   ),
-  ## Fig 2 combined--------------------------------------------
+  ### Fig combined--------------------------------------------
```

Comment:
Yeah -- I figured this was a slightly larger refactor and tbh I still know exactly what these refer to since we outlined them so long ago! I can change in a future PR but it will get more verbose to not use fig numbers. 

If you want me to change before zenodo, we need to align on figure order tm

Reply:
No need to change before zenodo. Maybe worth changing eventually either to reflect final figure order or not to state order explicitly

Reply:
Created #217